Changes

```diff
@@ -72,6 +72,9 @@ Working version
 
 ### Standard library:
 
+- #11837: Add `Array.mapi_inplace`.
+  (Léo Andrès, review by ?)
```

Comment:
You could fold this entry in the one below (#..., #...: Array.map_inplace, Array.mapi_inplace), and add Nicolás as a reviewer there.

Reply:
Good idea, and also mention `Float.Array.map_inplace`, `Float.Array.mapi_inplace`.